tests/test_frame.py

```diff
@@ -2598,15 +2598,15 @@ def test_align() -> None:
         columns=["A", "B", "C"],
     )
 
-    s0 = pd.Series(data={1: "1", 3: "3", 5: "5"})
+    s0 = pd.Series(data={0: "1", 3: "3", 5: "5"})
```

Comment:
Without this change, the first element is `np.nan`, so this allows us to see that the first element is a string 3 lines below.

Reply:
I don't understand how is `1` inferred as `np.nan`?

Reply:
In the code just below the above, we call `DataFrame.align()`.  The specified `DataFrame` does not have the index `1`, so the `align` call puts `np.nan` in the first element of the result.

Part of the reason this change was needed is because I changed the test to check the type of the `Series` to `Series[str]` and I wanted `check()` to check the type of the first element.